app/pb_kits/playbook/pb_checkbox/_checkbox.jsx

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+
+import React from 'react'
+
+type CheckboxProps = {
+  children: Array<React.ReactNode> | React.ReactNode,
```

Comment:
Will this component be using it's `children` props as well? I noticed it isn't currently in the rendered output.